KarhooUISDKTests/TestCases/Screens/BookingScreen/TripAllocation/KarhooTripAllocationPresenterSpec.swift

```diff
@@ -167,7 +167,7 @@ final class KarhooTripAllocationPresenterSpec: XCTestCase {
             mockTripService.trackTripCall.triggerPollSuccess(newTripUpdate)
 
             XCTAssertEqual(mockTripAllocationView.tripAllocatedCalled!.tripId, newTripUpdate.tripId)
-            XCTAssertFalse(mockTripService.trackTripCall.hasObserver)
+            XCTAssertTrue(mockTripService.trackTripCall.hasObserver)
```

Comment:
Why do these need changing?

Reply:
It fails on `develop` branch

<img width="407" alt="Screenshot 2021-02-16 at 10 19 19" src="https://user-images.githubusercontent.com/12498117/108049691-86664800-7040-11eb-971e-d98f4a8fad36.png">